File: Java Programs/SortingAlgos/BubbleSort.java
import java.util.*;

class BubbleSort {
    public static void main(String[] args) {
        int arr[] = {7,8,3,1,2};

        // Bubble sort
        for(int i = 0; i < arr.length - 1; i++) {
            for(int j = 0; j<arr.length - 1 - i; j++) {
                if(arr[j] > arr[j+1]){
                    // Swap the elements
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        }

        for(int i = 0; i < arr.length; i++){
            System.out.print(arr[i] + " ");
        }
        
    }
}

File: Python/Sorting Algos/BubbleSort.py
def bubbleSort(arr):
    for i in range(len(arr) - 1):
        for j in range(len(arr) - 1 - i):
            if(arr[j] > arr[j+1]):
                # Swap
                temp = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = temp

    return arr

if __name__ == '__main__':
    arr = [7,8,1,3,2,0,-1,1,-4]
    print(bubbleSort(arr))
